nearcore/src/state_sync.rs

```diff
@@ -594,7 +606,7 @@ fn get_latest_epoch(
     shard_id: &ShardId,
     chain: &Chain,
     epoch_manager: Arc<dyn EpochManagerAdapter>,
-) -> Result<(EpochId, EpochHeight, CryptoHash), Error> {
+) -> Result<(EpochId, EpochId, EpochHeight, CryptoHash), Error> {
```

Comment:
nit: I would consider a dedicated struct for this, especially given you habe two values with the same type.